tests/integration/conftest.py

```diff
@@ -133,6 +134,17 @@ def proxy(service, application, api_backend) -> Proxy:
     return proxy
 
 
+@pytest.fixture(scope='module')
+def backendservice(service, backend, application) -> BackendService:
```

Comment:
would suggest the name `backend_service`

Reply:
I've renamed it. BackendService(s)->BackendUsage(s) and backendservice(s)->backend_usage(s)